docs/src/compatibility.md

```diff
@@ -56,6 +56,9 @@ Example = "0.2.1"
 
 results in a versionbound on Example as `[0.2.1, 0.3.0)`.
 
+In particular, the depended package (`Example` in above example) may set `version = "0.2.4"` when it has feature additions
```

Comment:
Is "depended package" a thing one can say? Maybe , "the package depended on"?

Reply:
OK, this was something that was not super clear to me. Maybe it's better to say ``the upstream package (`Example` in above example)``?

Reply:
or just say `a package`?

Reply:
Sounds good to me. I changed it. (I tried to stick with the view from the downstream package since it's about `[compat]`. But I guess that was unnecessary consideration.)